lms/views/basic_lti_launch.py

```diff
@@ -37,8 +37,20 @@ def __init__(self, context, request):
         self.context = context
         self.request = request
 
-        # Configure the front-end mini-app to run.
-        self.context.js_config.update({"mode": "basic-lti-launch"})
+        self.context.js_config.update(
+            {
+                # Configure the front-end mini-app to run.
+                "mode": "basic-lti-launch",
+                # Add debug information (currently used in the gherkin tests)
+                "debug": {
+                    "tags": [
+                        "role:instructor"
+                        if request.lti_user.is_instructor
+                        else "role:learner"
```

Comment:
Without this there is no way to determine the different between an instructor and learner unless grading is enabled (which it isn't in the LTI tests we've implemented).